feature/google/src/main/java/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -469,8 +471,8 @@ class BillingWrapper(
                 var type: ProductType?
                 var presentedOffering: String?
                 synchronized(this@BillingWrapper) {
-                    type = productTypes[purchase.sku]
-                    presentedOffering = presentedOfferingsByProductIdentifier[purchase.sku]
+                    type = productTypes[purchase.skus[0]]
+                    presentedOffering = presentedOfferingsByProductIdentifier[purchase.skus[0]]
```

Comment:
super minor, but since this is an assumption that will break as soon as multi-line subscriptions are released, perhaps we could make that more explicit for future reference? 
i.e.: 
```kotlin
// Important: until multi-line subscriptions are released, 
// we assume that there will be only a single sku in the purchase. https://www.youtube.com/watch?v=iBxYrEkpCac&t=431s
val sku = purchase.skus[0]

type = productTypes[sku]
presentedOffering = presentedOfferingsByProductIdentifier[sku]
```
Ideally we would use our own rev.cat link instead of the youtube one in case they take it down or something

Reply:
i wonder if we could also put that assumption somewhere higher, like at the class level?

Reply:
maybe as an extension function?